src/com/jsonnetplugin/JsonnetIdentifierReference.java

```diff
@@ -150,7 +165,7 @@ private boolean findComprehensionBinding(List<ResolveResult> results, String ide
     @Override
     public PsiElement resolve() {
         ResolveResult[] resolveResults = multiResolve(false);
-        return resolveResults.length == 1 ? resolveResults[0].getElement() : null;
+        return resolveResults.length == 1 ? resolveResults[resolveResults.length-1].getElement() : null;
```

Comment:
I don't actually know what this change does; might not be needed since we're doing the "resolve to single result" thing in `multiResolve` itself